src/main/webapp/WEB-INF/templates/views/privileged/my_projects.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/head.html :: head"></head>
+<body>
+<div class="container-fluid">
+    <div class="container">
+        <h1>Welcome <span th:text="${Email}"></span></h1>
```

Comment:
chceme tu neskor dorobit aby mu zobrazilo meno alebo to nechame pod mailom?

Reply:
To je na vas ja som spravil ready ukazku. Neviem co ste si tam stanovili.